Resources/views/PageAdmin/select_site.html.twig

```diff
@@ -25,19 +25,23 @@ file that was distributed with this source code.
     {{ 'message_create_page'|trans({}, 'SonataPageBundle')}}
     </p>
 
-    <ul>
-        {% for site in sites %}
-            <li>
-                {% if current and site.id == current.id %}
-                    <img src="{{ asset('bundles/sonataadmin/famfamfam/accept.png') }}">
-                {% endif %}
-
-                <a href="{{ admin.generateUrl('create', {'siteId': site.id, 'uniqid': admin.uniqid}) }}">
-
-                {{ site.name }}
-                </a>
-                - {{ site.url }}
-            </li>
-        {% endfor%}
-    </ul>
+    {% if sites and sites|length > 0 -%}
```

Comment:
`if sites|default([])|length > 0` would be shorter, but why is not sites an empty array in the first place?